contracts/MANACrowdsale.sol

```diff
@@ -41,16 +40,22 @@ contract MANACrowdsale is ContinuousCrowdsale, WhitelistedCrowdsale, CappedCrowd
     {
         require(_rateStepDecrease > 0);
         require(_preferentialRate > 0);
-        require(_rate > _rateStepDecrease * (_endBlock - _startBlock));
+        require(_rate > _rateStepDecrease * (1 + _endBlock - _startBlock));
 
         rateStepDecrease = _rateStepDecrease;
         preferentialRate = _preferentialRate;
+
+        createContinuousSaleContract();
```

Comment:
The contract created here should be assigned to the `continuousSale` field. (credit to @frangio)